sklearn/svm/base.py

```diff
@@ -482,8 +482,8 @@ def _validate_for_predict(self, X):
     @property
     def coef_(self):
         if self.kernel != 'linear':
-            raise ValueError('coef_ is only available when using a '
-                             'linear kernel')
+            raise AttributeError('coef_ is only available when using a '
```

Comment:
For similar reasons we have defined `class NotFittedError(ValueError, AttributeError):`!

I suspect this is not a major concern as we don't expect this to be handled programmatically as a `ValueError`... but of course users' expectations may differ...